pgcli/packages/pgspecial.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import print_function
 import sys
 import logging
+import click
```

Comment:
Pulling in click as a dependency seems wrong to me. I'll explain the reasons in a longer comment.